packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -43,30 +46,29 @@ export class EditorPage {
 	 * Constructs an instance of the component.
 	 *
 	 * @param {Page} page The underlying page.
+	 * @param param0 Keyed object parameter.
+	 * @param param0.target Site type. Defaults to 'simple'.
 	 */
-	constructor( page: Page ) {
+	constructor( page: Page, { target = 'simple' }: { target?: 'simple' | 'atomic' } = {} ) {
```

Comment:
Once https://github.com/Automattic/wp-calypso/pull/61783 is merged I will import the `target` type instead.

Reply:
It's currently called `SiteType`, and it's defined in https://github.com/Automattic/wp-calypso/pull/61783/files#diff-9002489e27625a1d0b7fa59a8f40b5fdafe17d31b263fa5511df77ab88f3edacR3, but I think `Target` better communicates what it's about 👍🏻 

Right now these types are all specific to the `get-test-account-by-feature` module, but if we're going to use them in other places, we might as well move them to their own `.d.ts` file. What do you think?

Reply:
An `export` in front of the definition would also work. At some point there should be an effort to group together types, as we typically define and consume types in the same class and whenever it's needed externally the `export` keyword is prepended and we call it a day.